### ISSUE TITLE
Printing the parts

### ISSUE THREAD

**Issue by michaelhotker** [BCN3D/BCN3D-Moveo#37]
do any of the parts require supports at all? I could find anywhere in the manual that said anything about it.

**Comment by AlfonsoCalero** [comment]
I don't think you will find an instruction manual at such a low level. There are many parts of the project that will go on your own, although you can always count on people who have carried out the project and will lend you a hand. Regarding the subject of supports, you are supposed to have knowledge about whether an object should use supports or not when printing it.

**Comment by DominicM1213** [comment]
Has anyone figured out how to print the 1M1 part? I can't seem to get this lip to print properly.

**Comment by maedrone** [comment]
Yo lo. He impreso con soportes y me ha quedado muy bien, sinceramente no utilice ninguna configuración modificada, solo agregue soportes

**Comment by tkircher** [comment]
It's clear that whoever designed these parts doesn't understand design for 3D printing. Essentially all of these parts require complex supports, when a few small changes would make them require no supports. (For example, tapered hex nut seats instead of flat ones, which is 3D printing design 101)
